guides/common/modules/ref_cli-help.adoc

```diff
@@ -16,11 +16,4 @@ For more information, run `{foreman-installer} --full-help` on your {ProjectServ
 
 [[Hammer_CLI]]
 Hammer CLI:: You can manage {Project} on the command line using `hammer`.
-For more information on using Hammer CLI,
-ifdef::orcharhino[]
-see xref:sources/installation_and_maintenance/using_hammer_cli.adoc[Using Hammer CLI] or
-endif::[]
-ifdef::satellite[]
-see {HammerDocURL}[{HammerDocTitle}] or
-endif::[]
-run `hammer --help` on your {ProjectServer} or {SmartProxyServer}.
+For more information on using Hammer CLI, see {HammerDocURL}[{HammerDocTitle}] or run `hammer --help` on your {ProjectServer} or {SmartProxyServer}.
```

Comment:
```suggestion
For more information on using Hammer CLI, see {HammerDocURL}[_{HammerDocTitle}_] or run `hammer --help` on your {ProjectServer} or {SmartProxyServer}.
```
I'd prefer guide titles to be in italics, but it's not a blocker.

Reply:
Applied.